tests/Console/ConfigurationResolverTest.php

```diff
@@ -954,6 +954,94 @@ public function testResolveRulesWithOption(): void
         );
     }
 
+    /**
+     * Test the renamed rules.
+     *
+     * @dataProvider provideRenamedRulesCases
+     */
+    public function testResolveRenamedRulesWithUnknownRules(string $expectedMessage, string $renamedRule): void
+    {
+        $this->expectException(
+            \PhpCsFixer\ConfigurationException\InvalidConfigurationException::class
+        );
+        $this->expectExceptionMessage($expectedMessage);
+
+        $resolver = $this->createConfigurationResolver(['rules' => $renamedRule]);
+
+        $resolver->getRules();
+    }
+
+    public function provideRenamedRulesCases()
+    {
+        return [
+            [
+                'The rules contain unknown fixers: "blank_line_before_return" is a renamed rule,  did you mean "blank_line_before_statement" (note: use configuration "[\'statements\' => [\'return\']]")?'.PHP_EOL.PHP_EOL.
```

Comment:
```suggestion
                'The rules contain unknown fixers: "blank_line_before_return" is a renamed rule, did you mean "blank_line_before_statement" (note: use configuration "[\'statements\' => [\'return\']]")?'.PHP_EOL.PHP_EOL.
```

Reply:
Done in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5887/commits/279782fd05d2359a8bb0ae0c12244b6b103b31c6